src/test/scala/io/archivesunleashed/app/WriteGEXFTest.scala

```diff
@@ -57,7 +57,7 @@ class WriteGEXFTest extends FunSuite with BeforeAndAfter{
     assert(lines(testLines._4) == """</edges>""")
   }
 
-  test("creates the file from Array[Row]") {
+  test("Creates the gexf file from Array[Row]") {
```

Comment:
If we want to be pedantic we could rename to `GEXF` for consistency with `HTML` etc?